RInAppMessaging/Classes/IAMPreferenceBuilder.swift

```diff
@@ -27,6 +27,10 @@
 
     @objc @discardableResult
     public func build() -> IAMPreference {
+        if BundleInfo.applicationId?.starts(with: "jp.co.rakuten") == true, Bundle.tests == nil {
+            assert(preference.accessToken == nil || preference.accessToken != nil && preference.userId != nil,
```

Comment:
Nice idea to add this.
Can you confirm that this assert is only executed in a debug build?

Reply:
Can I confirm by definition?
https://developer.apple.com/documentation/swift/1541112-assert
`...active during testing but do not impact performance of shipping code.`